src/daria/analysis/concentrationanalysis.py

```diff
@@ -216,24 +254,33 @@ def find_cleaning_filter(
             baseline_images (list of daria.Image): series of baseline_images.
             reset (bool): flag whether the cleaning filter shall be reset.
         """
+        print("Setting up a cleaning filter.")
```

Comment:
Should this be within some verbosity check? I guess it is not needed, but I always find it nice to be able to turn of printing.

Reply:
I can also remove them. Setting up a cleaning filter takes quite some time (sometimes minutes). I will adjust such that the printing only happens when it takes long time.

Reply:
Sounds good!